setup.cfg

```
@@ -40,7 +40,7 @@ scripts = django/bin/django-admin.py
 include_package_data = true
 zip_safe = false
 install_requires =
-    asgiref >= 3.2
+    asgiref ~= 3.2.10
```

I will forwardport this to master with:
```suggestion
    asgiref >= 3.2.10
```